d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -108,7 +113,6 @@ def on_finish(self, finish_info):
 
 rest_market = RestMarketClient(simulation_id, area_uuid, domain_name, websocket_domain_name)
 market_slot_string = today().add(minutes=60).format(DATE_TIME_FORMAT)
```

Comment:
you can remove these lines too.
the latter is also a remnant of the former API for the last_market_stats where the user could acquire the stats of a specific market. Thanks!